src/server/api/boards/boards.js

```diff
@@ -98,8 +107,27 @@ function getBoardById(username, boardId, callback) {
   });
 }
 
+function deleteBoardById(username, boardId, callback) {
```

Comment:
i think the `boardId` would be enough because there will be no duplicates but its not a fatal problem.